Formula/gnu-tar.rb

```diff
@@ -11,6 +11,14 @@ class GnuTar < Formula
     sha256 "1a559b78e6f1a6594b18a9ba2aa2e9828af2736aacc4aec07911fe7638e80e68" => :el_capitan
   end
 
+  head do
+    url "git://git.savannah.gnu.org/tar.git"
```

Comment:
Should be able to use `https://` if you write it this way:

```
https://git.savannah.gnu.org/git/tar.git
```

Reply:
@DomT4, sure, but threre was error in clone at time of PR.